packages/esm-messages-app/src/config-schema.ts

```diff
@@ -0,0 +1,13 @@
+import { Type } from '@openmrs/esm-framework';
+
+export const configSchema = {
+  endOfMessage: {
+    _type: Type.Number,
+    _description: 'Specifies the end of message',
+    _default: [],
+  },
```

Comment:
Could we have a better description of this? I have no idea what I would set this to or why I would use this.

Reply:
@ibacher   in the 2.x (connect for life backend) sets the number of times a message is sent to the selected user before it stops

Reply:
suggested description 'specifies times to end of message service'

Reply:
more like number of times the patient's subscribed message service is sent out ,after 7 times it deactivates (came as an implementation of CFL)

Reply:
If this is something built in to the backend module, it _really_ shouldn't be configured through the frontend configuration. The goal is to have one place to configure things. I'm still confused about what the heck this actually configures, and while I'm probably just being dense, it doesn't seem like a good idea to have a configuration option that's hard to explain.